README.md

```diff
@@ -193,6 +193,9 @@ Here are the build in dances and their options
 + borderWidth
   + min : Minimum value given to `border-width`. Default: `0`
   + max : Maximum value given to `borderr-width`. Default: `5`
++ fontWidth
```

Comment:
You named it `fontSize` in the code ;)

Reply:
Ok, gonna fix it 🤣 

Did you see the borderr just above ? Do you want me to also fix it ?

Reply:
Oh ! Didn't see this one, if you want to fix it, i would prefer to have a separated PR for that fix if you don't mind ;)

Reply:
Ok I'm gonna PR Farm 🤠